compiler/benches/executor_benchmark.rs

```diff
@@ -34,6 +42,26 @@ fn criterion_benchmark(c: &mut Criterion) {
             run_witgen(
                 &pil_with_constants.pil,
                 pil_with_constants.fixed_cols.clone(),
+                vec![],
+            )
+        })
+    });
+
+    // The first chunk of `many_chunks`, with Poseidon co-processor & bootloader
```

Comment:
why just the first chunk?

Reply:
I think that's enough to test witness generation (already takes > 7min to run it 10 times in the benchmark). The use-case is more that witgen is actually doing something for all of the rows.